docs/formats.md

```diff
@@ -33,7 +33,7 @@ There is an extensive (but not exhaustive) list of [included formats](#pre-defin
 
 ### Format configuration
 
-Formats can take configuration to make them more flexible. This allows you to re-use the same format multiple times with different configurations or to allow the format to use data not defined in the tokens themselves. To configure a format, add extra attributes on the file object in your configuration like the following:
+Formats can take configuration to make them more flexible. This allows you to re-use the same format multiple times with different configuration or to allow the format to use data not defined in the tokens themselves. To configure a format add extra attributes on the file object in your configuration as written below:
```

Comment:
I like the original better, actually, thought the first sentence is confusing. I do like your change at the end to "as written below". So maybe this:

Formats are flexible and may accept configuration options. This allows you to re-use the same format multiple times with different configurations or to allow the format to use data not defined in the tokens themselves. To configure a format, add extra attributes on the file object in your configuration as written below:

@dbanksdesign thoughts?

Reply:
love it :shipit: